src/scss/lr7-classic-theme/variables/_toolbars.scss

```diff
@@ -0,0 +1,22 @@
+// Basics of a toolbar
+$toolbar-height: 48px;
+
+//== Toolbar Default
+$toolbar-bottom-border: darken(#F5F8FA, 4);
+$toolbar-default-bg: #F5F8FA;
```

Comment:
These toolbar-default variables should be prefixed with management-toolbar instead of toolbar-default.